<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s</class>
 <widget class="QWidget" name="s">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMI Open Data - Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1032</width>
        <height>274</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_metObs">
            <property name="title">
             <string>API Key MetObs</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLineEdit" name="DMI_apiKey_metObs">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Enter API Key for MetObs</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="DMI_apiKey_oceanObs">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Enter API Key for OceanObs</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="DMI_apiKey_climateData">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Enter API Key for Climate Data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="DMI_apiKey_lightningData">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Enter API Key for Lightning Data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="DMI_apiKey_radarData">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Enter API Key for Radar Data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://confluence.govcloud.dk/display/FDAPI/User+Creation#UserCreation-FindYourAPIKeyRetrieveyourAPIKey&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Find your API Key for your desired API&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Not a user of DMI Open data? &lt;a href=&quot;https://confluence.govcloud.dk/display/FDAPI/User+Creation#UserCreation-UserCreation&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Create user guide&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
